docs/changes.rst

```diff
@@ -54,13 +58,33 @@ Released: not yet
 * Fixed that the added setup.py commands (test, leaktest, installtest) were not
   displayed. They are now displayed at verbosity level 1 (using '-v').
 
+* Fixed that the sending of CIM request messages was retried in case no
+  response was received within the timeout, or in case of the 'status' and
+  'other' conditions of the "requests" package.
+  Retries are now only performed during connection setup, but not for sending
```

Comment:
DONE - improved that by changing the entry to:
```
* Previously, the sending of CIM request messages was retried in case no
  response was received within the timeout. This could potentially have resulted
  in executing operations multipe times. That is an issue for non-idempotent
  operations such as instance creation/deletion or method invocation. Fixed
  that by retrying now only during connection setup, but not for the sending
  of CIM request messages. (issue #2853)
```